src/rgw/rgw_reshard.cc

```diff
@@ -527,14 +527,10 @@ int RGWBucketReshard::do_reshard(int num_shards,
 				 ostream *out,
 				 Formatter *formatter)
 {
-  rgw_bucket& bucket = bucket_info.bucket;
-
-  int ret = 0;
-
-  if (out) {
-    (*out) << "tenant: " << bucket_info.bucket.tenant << std::endl;
-    (*out) << "bucket name: " << bucket_info.bucket.name << std::endl;
-    (*out) << "old bucket instance id: " << bucket_info.bucket.bucket_id <<
+  if (const rgw_bucket& bucket = bucket_info.bucket; out) {
```

Comment:
this declaration doesn't belong in the if(), please move it down onto a line of its own

Reply:
@cbodley updated and repushed